js/script.js

```diff
@@ -768,6 +792,10 @@ function Add_EventHandlers_To_Document()
             }
 
         }
+        if(e.code === "Delete")
```

Comment:
I have a Mac, so I had to press `fn` then my `Delete` key to hit this if statement.

Can we add `Backspace` to this condition as well, so that users can always press one key?

```javascript
if(e.code === "Delete" || e.code === "Backspace")
{
    ...
}
```

@Luca-Ilari  WDYT? Any edge cases we're not considering?